app_dart/lib/src/request_handlers/get_status.dart

```diff
@@ -30,15 +31,30 @@ class GetStatus extends RequestHandler<Body> {
   final DatastoreServiceProvider datastoreProvider;
   final BuildStatusProvider buildStatusProvider;
 
+  static const String lastCommitKeyParam = 'lastCommitKey';
+
   @override
   Future<Body> get() async {
+    final String encodedLastCommitKey =
+        request.uri.queryParameters[lastCommitKeyParam];
+    final DatastoreService datastore = datastoreProvider();
+    final KeyHelper keyHelper = config.keyHelper;
+    final int commitNumber = config.commitNumber;
+
+    /// [timestamp] is initially set as the current time, which allows query return
+    /// latest commit list. If [owerKeyParam] is not empty, [timestamp] will be set
+    /// as the timestamp of that [commit], and the query will return lastest commit
+    /// list whose timestamp is smaller than [timestamp].
```

Comment:
This documentation should be moved to document `_obtainTimestamp`.

Reply:
moved.